src/routes/+layout.svelte

```diff
@@ -82,8 +89,20 @@
                 loading.set(false);
             }
         }
+
+        /**
+         * typesafe-i18n here
+         */
+
+        const detectedLocale = detectLocale(localStorageDetector);
+        await chooseLocale(detectedLocale);
     });
 
+    const chooseLocale = async (locale: Locales) => {
```

Comment:
Is this only used above? If so, we probably don't need a separate function

Reply:
done :D